src/components/BulkEditList/BulkEditListFilters/QueryTextArea/QueryTextArea.js

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Button,
+  TextArea,
+} from '@folio/stripes/components';
+
+import { ResetButton } from '@folio/stripes-acq-components';
+
+
+export const QueryTextArea = ({ filters, setQueryText }) => {
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  useEffect(() => (filters.queryText === '' ? setIsDisabled(true) : setIsDisabled(false)),
+    [filters.queryText]);
+
+  const resetButtonHandler = () => {
+    setQueryText({
+      ...filters, queryText: '',
```

Comment:
Filters doesn't required. You should use `prevState => {}` signature in `setQueryText`

Reply:
Done. Thanks for that.